documentation/jetty-documentation/src/main/asciidoc/operations-guide/modules/module-eex-webapp.adoc

```diff
@@ -0,0 +1,25 @@
+//
+// ========================================================================
+// Copyright (c) 1995-2022 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+[[og-module-eex-webapp]]
+===== Module `ee{8,9,10}-webapp`
```

Comment:
Can we parametrize this recurring `ee{8,9,10}` into a variable in `operations-guide/.asciidoctorconfig`?

I think all the variables so far are lower case except `JETTY_HOME`... so perhaps convert to `jetty-home` the variable name, and introduce `ee-all` and `ee-current` variables?

Reply:
This is done, but I'm going do the conversion of "JETTY_HOME" to lower-case in a separate pull request since it will touch a lot of files.